diff -urN aa/dirac-0.4.1/configure.ac dirac-0.4.1/configure.ac
--- aa/dirac-0.4.1/configure.ac	2004-08-26 13:02:52.000000000 +0200
+++ dirac-0.4.1/configure.ac	2004-08-26 15:38:38.714554768 +0200
@@ -66,7 +66,7 @@
 		use_msvc=yes
 		;;
 	*)
-		AC_PROG_RANLIB
+		AC_PROG_LIBTOOL
 		;;
 esac
 AM_CONDITIONAL(USE_MSVC, test x"$use_msvc" = "xyes")
diff -urN aa/dirac-0.4.1/decoder/Makefile.am dirac-0.4.1/decoder/Makefile.am
--- aa/dirac-0.4.1/decoder/Makefile.am	2004-08-25 18:58:10.000000000 +0200
+++ dirac-0.4.1/decoder/Makefile.am	2004-08-26 15:38:59.028466584 +0200
@@ -4,7 +4,7 @@
 
 dirac_decoder_SOURCES = decmain.cpp
 
-LDADD = ../libdirac_decoder/libdirac_decoder.a ../libdirac_common/libdirac_common.a
+LDADD = ../libdirac_decoder/libdirac_decoder.la ../libdirac_common/libdirac_common.la
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
diff -urN aa/dirac-0.4.1/encoder/Makefile.am dirac-0.4.1/encoder/Makefile.am
--- aa/dirac-0.4.1/encoder/Makefile.am	2004-08-25 18:58:11.000000000 +0200
+++ dirac-0.4.1/encoder/Makefile.am	2004-08-26 15:39:22.011972560 +0200
@@ -4,7 +4,7 @@
 
 dirac_encoder_SOURCES = encmain.cpp
 
-LDADD = ../libdirac_encoder/libdirac_encoder.a ../libdirac_common/libdirac_common.a ../libdirac_motionest/libdirac_motionest.a
+LDADD = ../libdirac_encoder/libdirac_encoder.la ../libdirac_common/libdirac_common.la ../libdirac_motionest/libdirac_motionest.la
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
diff -urN aa/dirac-0.4.1/libdirac_common/Makefile.am dirac-0.4.1/libdirac_common/Makefile.am
--- aa/dirac-0.4.1/libdirac_common/Makefile.am	2004-08-25 18:58:11.000000000 +0200
+++ dirac-0.4.1/libdirac_common/Makefile.am	2004-08-26 15:40:09.227794664 +0200
@@ -10,9 +10,9 @@
               mv_codec.cpp pic_io.cpp upconvert.cpp wavelet_utils.cpp \
               cmd_line.cpp dirac_assertions.cpp
 
-lib_LIBRARIES = libdirac_common.a
+lib_LTLIBRARIES = libdirac_common.la
 
-libdirac_common_a_SOURCES = $(h_sources) $(cpp_sources)
+libdirac_common_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)/libdirac_common
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/libdirac_decoder/Makefile.am dirac-0.4.1/libdirac_decoder/Makefile.am
--- aa/dirac-0.4.1/libdirac_decoder/Makefile.am	2004-08-25 18:58:12.000000000 +0200
+++ dirac-0.4.1/libdirac_decoder/Makefile.am	2004-08-26 15:45:19.620607784 +0200
@@ -6,9 +6,9 @@
 cpp_sources = comp_decompress.cpp frame_decompress.cpp seq_decompress.cpp \
              dirac_cppparser.cpp dirac_parser.cpp
 
-lib_LIBRARIES = libdirac_decoder.a
+lib_LTLIBRARIES = libdirac_decoder.la
 
-libdirac_decoder_a_SOURCES = $(h_sources) $(cpp_sources)
+libdirac_decoder_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)/libdirac_decoder
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/libdirac_encoder/Makefile.am dirac-0.4.1/libdirac_encoder/Makefile.am
--- aa/dirac-0.4.1/libdirac_encoder/Makefile.am	2004-08-25 18:58:12.000000000 +0200
+++ dirac-0.4.1/libdirac_encoder/Makefile.am	2004-08-26 15:45:42.139184440 +0200
@@ -4,9 +4,9 @@
 
 cpp_sources = comp_compress.cpp frame_compress.cpp quality_monitor.cpp seq_compress.cpp
 
-lib_LIBRARIES = libdirac_encoder.a
+lib_LTLIBRARIES = libdirac_encoder.la
 
-libdirac_encoder_a_SOURCES = $(h_sources) $(cpp_sources)
+libdirac_encoder_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)/libdirac_encoder
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/libdirac_motionest/Makefile.am dirac-0.4.1/libdirac_motionest/Makefile.am
--- aa/dirac-0.4.1/libdirac_motionest/Makefile.am	2004-08-25 18:58:12.000000000 +0200
+++ dirac-0.4.1/libdirac_motionest/Makefile.am	2004-08-26 15:44:00.462641624 +0200
@@ -4,9 +4,9 @@
 
 cpp_sources = block_match.cpp downconvert.cpp me_mode_decn.cpp me_subpel.cpp me_utils.cpp motion_estimate.cpp pixel_match.cpp
 
-lib_LIBRARIES = libdirac_motionest.a
+lib_LTLIBRARIES = libdirac_motionest.la
 
-libdirac_motionest_a_SOURCES = $(h_sources) $(cpp_sources)
+libdirac_motionest_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)/libdirac_motionest
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/picheader/Makefile.am dirac-0.4.1/picheader/Makefile.am
--- aa/dirac-0.4.1/picheader/Makefile.am	2004-08-25 18:58:12.000000000 +0200
+++ dirac-0.4.1/picheader/Makefile.am	2004-08-26 15:45:58.186744840 +0200
@@ -4,7 +4,7 @@
 
 make_header_SOURCES = headmain.cpp
 
-LDADD = ../libdirac_common/libdirac_common.a
+LDADD = ../libdirac_common/libdirac_common.la
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
diff -urN aa/dirac-0.4.1/unit_tests/Makefile.am dirac-0.4.1/unit_tests/Makefile.am
--- aa/dirac-0.4.1/unit_tests/Makefile.am	2004-08-25 18:58:13.000000000 +0200
+++ dirac-0.4.1/unit_tests/Makefile.am	2004-08-26 15:45:36.198087624 +0200
@@ -8,7 +8,7 @@
 						 arrays_test.h \
 						 arrays_test.cpp
 
-LDADD = ../libdirac_encoder/libdirac_encoder.a ../libdirac_common/libdirac_common.a ../libdirac_motionest/libdirac_motionest.a -ldl -lcppunit
+LDADD = ../libdirac_encoder/libdirac_encoder.la ../libdirac_common/libdirac_common.la ../libdirac_motionest/libdirac_motionest.la -ldl -lcppunit
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
diff -urN aa/dirac-0.4.1/util/conversion/common/Makefile.am dirac-0.4.1/util/conversion/common/Makefile.am
--- aa/dirac-0.4.1/util/conversion/common/Makefile.am	2004-08-25 18:58:14.000000000 +0200
+++ dirac-0.4.1/util/conversion/common/Makefile.am	2004-08-26 15:43:27.154705200 +0200
@@ -4,9 +4,9 @@
 
 cpp_sources = setstdiomode.cpp bitmap.cpp
 
-lib_LIBRARIES = libconv_common.a
+lib_LTLIBRARIES = libconv_common.la
 
-libconv_common_a_SOURCES = $(h_sources) $(cpp_sources)
+libconv_common_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/util/conversion/Makefile.am dirac-0.4.1/util/conversion/Makefile.am
--- aa/dirac-0.4.1/util/conversion/Makefile.am	2004-08-25 18:58:13.000000000 +0200
+++ dirac-0.4.1/util/conversion/Makefile.am	2004-08-26 15:44:28.521376048 +0200
@@ -17,7 +17,7 @@
 UYVYtoRGB_SOURCES=UYVYtoRGB.cpp
 BMPtoRGB_SOURCES=BMPtoRGB.cpp
 
-LDADD = common/libconv_common.a
+LDADD = common/libconv_common.la
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
diff -urN aa/dirac-0.4.1/util/instrumentation/libdirac_instrument/Makefile.am dirac-0.4.1/util/instrumentation/libdirac_instrument/Makefile.am
--- aa/dirac-0.4.1/util/instrumentation/libdirac_instrument/Makefile.am	2004-08-25 18:58:15.000000000 +0200
+++ dirac-0.4.1/util/instrumentation/libdirac_instrument/Makefile.am	2004-08-26 15:45:03.377077176 +0200
@@ -4,9 +4,9 @@
 
 cpp_sources = overlay.cpp draw_overlay.cpp motion_colour.cpp motion_arrows.cpp motion_colour_arrows.cpp sad.cpp pred_mode.cpp split_mode.cpp overlay_symbols.cpp
 
-lib_LIBRARIES = libdirac_instrument.a
+lib_LTLIBRARIES = libdirac_instrument.la
 
-libdirac_instrument_a_SOURCES = $(h_sources) $(cpp_sources)
+libdirac_instrument_la_SOURCES = $(h_sources) $(cpp_sources)
 
 library_includedir=$(includedir)/$(GENERIC_LIBRARY_NAME)
 library_include_HEADERS = $(h_sources)
diff -urN aa/dirac-0.4.1/util/instrumentation/Makefile.am dirac-0.4.1/util/instrumentation/Makefile.am
--- aa/dirac-0.4.1/util/instrumentation/Makefile.am	2004-08-25 18:58:14.000000000 +0200
+++ dirac-0.4.1/util/instrumentation/Makefile.am	2004-08-26 15:44:51.066948600 +0200
@@ -8,7 +8,7 @@
 
 dirac_instrumentation_SOURCES = $(h_sources) $(cpp_sources)
 
-LDADD = ../../libdirac_encoder/libdirac_encoder.a ../../libdirac_common/libdirac_common.a ../../libdirac_motionest/libdirac_motionest.a libdirac_instrument/libdirac_instrument.a
+LDADD = ../../libdirac_encoder/libdirac_encoder.la ../../libdirac_common/libdirac_common.la ../../libdirac_motionest/libdirac_motionest.la libdirac_instrument/libdirac_instrument.la
 
 if USE_MSVC
 CLEANFILES = *.pdb *.ilk
